src/pages/components/select/usage.mdx

```diff
@@ -290,49 +294,48 @@ appropriately for its context.
 
 #### Label
 
+Concise labels for text and data inputs help users understand what information
+is being requested of them.
+
 - Labels inform users what to expect in the list of select options.
-- Keep the label short and concise by limiting it to a single line of text.
+- Labels are not helper text; be succinct. Use one to three words only.
 
 #### Helper text
 
 - Helper text is pertinent information that assists the user in choosing the
-  right selection from the select menu.
+  right selection from the select list.
 - Helper text is optional but replaced with warning or error text when these
   states appear.
 
-#### Order
+#### List order
 
-The order of the select list should be based on the frequency of use. If
-applicable, the list should be in alphabetical order or in increasing order
-relative to the content.
+If applicable, the list should be in alphabetical order or in increasing order
+relative to the content. Otherwise, the order of the select list should be based
+on the frequency of use.
 
 <Row>
 <Column colLg={8}>
 
-![Select menu order](images/select-menu-order.png)
+![Select list order](images/select-list-order.png)
 
 </Column>
 </Row>
 
-#### Overflow content
-
-Avoid having multiple lines of text per item in a select menu. The menu items
-should be no longer than three words.
-
 ## Behaviors
 
 ### States
 
 The select has multiple interactive states for both default and inline variants:
-**enabled**, **hover**, **focus**, **open**, **error**, **warning**,
-**disabled**, and **read-only**.
+**enabled**, **hover**, **selected**, **focus**, **open**, **error**,
```

Comment:
There could be a row in the table to represent and describe the `Open state` since it's mentioned in the paragraph above and in the image.